build.gradle.kts

```diff
@@ -74,7 +74,11 @@ sonarqube {
     }
 }
 
-extra["testcontainersVersion"] = "1.18.2"
+tasks.processTestAot {
+    enabled = false
```

Comment:
Why is this disabled?

Reply:
I ran into https://github.com/spring-projects/spring-boot/issues/36997. Sadly, there's no other workaround than disabling the task.